discovery-infra/install_cluster.py

```diff
@@ -155,6 +158,12 @@ def main():
         type=str,
         default='assisted-service'
     )
+    parser.add_argument(
+        '-cn',
+        '--cluster-name',
```

Comment:
still don't understand why we need name

Reply:
for cases you want to create two clusters that talk with the same assisted-service.